cmd/sti/main.go

```diff
@@ -126,7 +130,7 @@ func newCmdCreate() *cobra.Command {
 				cmd.Help()
 				os.Exit(1)
 			}
-			b := sti.NewCreate(args[0], args[1])
+			b := create.NewCreate(args[0], args[1])
```

Comment:
We should come up with some unified way of getting objects, here we're using `create.NewCreate`, whereas a couple of lines before you do `strategies.GetBuilder`. I'd propose to use the [origin convention](https://github.com/openshift/origin/blob/master/pkg/cmd/server/origin/master.go#L242-L249), so in above cases you'd have:
`strategies.New()`, `create.New()`, etc.